seqr/views/apis/anvil_workspace_api.py

```diff
@@ -174,6 +181,27 @@ def create_project_from_workspace(request, namespace, name):
         logger.error('Uploading sample IDs to Google Storage failed. Errors: {}'.format(str(ee)), request.user,
                      detail=sample_ids)
 
+    # trigger dags
+    anvil_type = 'AnVIL_{sample_type}'.format(sample_type=request_json['sampleType'])
+
+    anvil_variables = get_variables(anvil_type)
+    updated_anvil_variables = generate_AnVIL_variables(project, data_path, request_json['sampleType'])
+    projects = anvil_variables["active_projects"]
+    updated_projects = updated_anvil_variables["active_projects"]
+
+    if anvil_variables != updated_anvil_variables: 
+        update_variables(anvil_type, updated_anvil_variables)
+
+    dag_id = "seqr_vcf_to_es_{anvil_type}_v{version}".format(anvil_type=anvil_type, version=DAG_VERSION)
+
+    if set(projects) != set(updated_projects):
+        dag_task_ids = get_task_ids(dag_id)
+        updated_task_ids = get_task_ids(dag_id)
+        while(set(dag_task_ids) == set(updated_task_ids)):
```

Comment:
so this definitely works, but I do wonder if theres a more explicit way to do this then just seeing if anything changed. What do the task ids looks like, in the end?